addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/discovery/FreeboxDiscoveryService.java

```diff
@@ -142,6 +146,27 @@ public void onDataFetched(ThingUID bridge, LanHostsConfig hostsConfig) {
                 }
             }
         }
+
+        if (airPlayDevices != null) {
+            // AirPlay devices
+            for (AirMediaReceiver device : airPlayDevices) {
+                String name = device.getName();
+                Boolean videoCapable = device.isVideoCapable();
+                logger.debug("AirPlay Device name {} video capable {}", name, videoCapable);
+                if ((name != null) && !name.isEmpty() && Boolean.TRUE.equals(videoCapable)) {
```

Comment:
Yes I am not certain that this field will always be set by the API. I consider the null case by security.

Ok to use StringUtils.